h2o-algos/src/main/java/hex/glm/ComputationState.java

```diff
@@ -403,13 +425,84 @@ private static void fillSubRange(int N, int c, int [] ids, double [] src, double
   }
 
   public double [] betaMultinomial(){return _beta;}
+  
+  public double [] betaMultinomial(int c, double [] beta) {
+     return extractSubRange(_activeData.fullN()+1,c,_activeDataMultinomial[c].activeCols(),beta);
+   }
+
+  public double [] betaMultinomialFull(int c, double [] beta) {
+     if (_parms._remove_collinear_columns)
+      return extractSubRange(_betaLengthPerClass,c,_activeDataMultinomial[c].activeCols(),beta);
+     else
+       return extractSubRange(_activeData.fullN()+1,c,_activeDataMultinomial[c].activeCols(),beta);
```

Comment:
Seems to me that _betaLengthPerClass doesn't change and if !_rcc then I think _activeData.fullN() shouldn't change as well.

If that's so, could we simplify the code to use only `_betaLengthPerClass` so we can get rid of the if statement?

Reply:
actually _activeData.fullN() changes when inactive columns are removed.  I was confused with this thing in the beginning and it takes me a very long time to figure this one out......